### DISCUSSION TITLE
Add new configuration files for SlowFast 新版 config 适配——SlowFast

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#541] [MMAction2]
# Add new configuration files for SlowFast 
 新版 config 适配——SlowFast 

mmengine implements a new config mechanism, based on the new config mechanism, please update configuration file for adaptation. Target config path: configs/recognition/slowfast

mmengine 实现了一套新的 config 机制，基于新的 config 机制进行配置文件的适配
- 相关适配教程文档：[https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)  
该任务适配的 config 目录：configs/dgcnn/ ，完成所有文件适配

## Task Information

**Repository**：MMAction2

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：jts250

**Claim Time**：2023年10月25日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by jts250** [comment]
Claim the mission

**Reply by jts250** [reply]
https://github.com/open-mmlab/mmaction2/pull/2753

**Comment by jts250** [comment]
I have completed the modifications to the new config file, but I am unable to resolve these warnings when committing. How should I handle them? or should I ignore them?

Warning Details:
configs/dgcnn/recognition/slowfast/slowfast_r101_8xb8_8x8x1_256e_kinetics400_rgb.py:4:5: F403 'from .slowfast_r50_8xb8_8x8x1_256e_kinetics400_rgb import ' used; unable to detect undefined names
configs/dgcnn/recognition/slowfast/slowfast_r101_8xb8_8x8x1_256e_kinetics400_rgb.py:4:5: F401 '.slowfast_r50_8xb8_8x8x1_256e_kinetics400_rgb.' imported but unused
configs/dgcnn/recognition/slowfast/slowfast_r101_r50_32xb8_4x16x1_256e_kinetics400_rgb.py:4:5: F403 'from .slowfast_r50_8xb8_4x16x1_256e_kinetics400_rgb import ' used; unable to detect undefined names
configs/dgcnn/recognition/slowfast/slowfast_r101_r50_32xb8_4x16x1_256e_kinetics400_rgb.py:4:5: F401 '.slowfast_r50_8xb8_4x16x1_256e_kinetics400_rgb.' imported but unused
configs/dgcnn/recognition/slowfast/slowfast_r50_8xb8-8x8x1_steplr_256e_kinetics400_rgb.py:4:5: F403 'from .slowfast_r50_8xb8_8x8x1_256e_kinetics400_rgb import ' used; unable to detect undefined names
configs/dgcnn/recognition/slowfast/slowfast_r50_8xb8-8x8x1_steplr_256e_kinetics400_rgb.py:4:5: F401 '.slowfast_r50_8xb8_8x8x1_256e_kinetics400_rgb.' imported but unused
configs/dgcnn/recognition/slowfast/slowfast_r50_8xb8_4x16x1_256e_kinetics400_rgb.py:18:5: F403 'from ...base.default_runtime import *' used; unable to detect undefined names
configs/dgcnn/recognition/slowfast/slowfast_r50_8xb8_4x16x1_256e_kinetics400_rgb.py:19:5: F403 'from ...base.models.slowfast_r50 import *' used; unable to detect undefined names
configs/dgcnn/recognition/slowfast/slowfast_r50_8xb8_4x16x1_256e_kinetics400_rgb.py:130:1: F405 'default_hooks' may be undefined, or defined from star imports: ...base.default_runtime, ...base.models.slowfast_r50
configs/dgcnn/recognition/slowfast/slowfast_r50_8xb8_8x8x1_256e_kinetics400_rgb.py:4:5: F403 'from .slowfast_r50_8xb8_4x16x1_256e_kinetics400_rgb import ' used; unable to detect undefined names
configs/dgcnn/recognition/slowfast/slowfast_r50_8xb8_8x8x1_256e_kinetics400_rgb.py:4:5: F401 '.slowfast_r50_8xb8_4x16x1_256e_kinetics400_rgb.' imported but unused

**Comment by jts250** [comment]
https://github.com/open-mmlab/mmaction2/pull/2753
